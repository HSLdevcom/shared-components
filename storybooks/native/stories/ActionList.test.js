import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  framework: 'react-native',
  storyKindRegex: /^ActionList$/
});
