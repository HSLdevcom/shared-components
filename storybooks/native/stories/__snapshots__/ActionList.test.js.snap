// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots ActionList ActionListItem 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessibilityRole="button"
    active={false}
    centered={false}
    href={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "alignItems": "center",
          "borderBottomWidth": 3,
          "borderColor": "transparent",
          "borderStyle": "solid",
          "flexDirection": "row",
          "justifyContent": "flex-start",
        },
        Array [
          Object {},
          Array [
            Object {},
            undefined,
          ],
        ],
      ]
    }
  >
    <Text
      accessibilityRole="heading"
      accessible={true}
      allowFontScaling={true}
      aria-level="4"
      arrow={true}
      centered={false}
      ellipsizeMode="tail"
      icon={false}
      style={
        Array [
          Object {
            "color": "#333333",
            "fontFamily": "Gotham Rounded SSm A, Gotham Rounded SSm B, Arial, Georgia, serif",
            "fontSize": 12.25,
            "fontWeight": "500",
          },
          Array [
            Object {
              "color": "#007ac9",
              "flexGrow": 2,
              "flexWrap": "wrap",
              "fontSize": 18.375,
              "fontWeight": "500",
              "marginRight": 13.125,
              "marginVertical": 15.75,
            },
            undefined,
          ],
        ]
      }
    >
      Arvoliput matkakortilla
    </Text>
    <Svg
      height={13}
      preserveAspectRatio="xMidYMid meet"
      version="1.1"
      viewBox="0 0 17 36"
      width={8}
    >
      <G
        originX={8.5}
        originY={17.5}
        scale={-1}
      >
        <Path
          d="M13.846,0.991 C15.366,0.991 16.486,2.111 16.486,3.551 C16.566,4.271 16.246,4.751 15.846,5.391 L6.326,18.351 L15.766,31.072 C16.166,31.552 16.486,32.272 16.486,32.912 C16.486,34.352 15.366,35.552 13.926,35.552 C12.966,35.552 12.166,34.992 11.526,34.192 L0.965,20.431 C0.325,19.631 0.005,18.991 0.005,18.271 C0.005,17.471 0.325,16.911 0.965,16.111 L11.686,2.191 C12.166,1.471 12.886,0.991 13.846,0.991 Z"
          fill="#007ac9"
          id="Shape"
        />
      </G>
    </Svg>
  </View>
</View>
`;

exports[`Storyshots ActionList default 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
/>
`;
